deps/npm/node_modules/request/node_modules/qs/Readme.md

```diff
@@ -0,0 +1,233 @@
+# qs
```

Comment:
Hmm... after last week's goat rodeo, I reviewed my got settings, and I'm not actually doing anything special with case sensitivity when creating the patch. It's probably something to do with removing `deps/npm` completely before unpacking the new version, but that's important to do to catch all of the deletions. I'll investigate and see what I can come up with. I don't think that creating the patch on a case-sensitive filesystem is going to help when applying the patch on a case-insensitive filesystem.